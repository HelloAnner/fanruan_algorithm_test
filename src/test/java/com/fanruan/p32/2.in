5
28765
10136
23277
28513
1090
