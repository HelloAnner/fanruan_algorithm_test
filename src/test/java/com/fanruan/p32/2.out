287652851323277109010136